app/templates/components/labs-map.hbs

```diff
@@ -0,0 +1,47 @@
+{{!--
+  This template overrides the template for an addon, ember-mapbox-composer.
+  e-mapbox-composer uses private APIs in e-mapbox-gl. Those changed, so we need to
+  manually configure the interface here.
+--}}
+
+{{#if _loader.isLoaded}}
```

Comment:
any reason not to collapse these into a single (and)-ed condition?

Reply:
@julialucyhogan labs-map irresponsibly extends the mapbox-gl, but you cannot extend templates. This is ported from that addon (one our team does not own) so I brought it into our source code.